jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwJoystickInput.java

```diff
@@ -57,6 +57,8 @@
 
     private final Map<JoystickButton, Boolean> joyButtonPressed = new HashMap<>();
 
+    private InputManager inputManager;
```

Comment:
FYI `private RawInputListener listener;` in line 54  can be cast to InputManager also.